Dockerfile

```diff
@@ -41,6 +41,18 @@ RUN cd ./src/frontend && npm install --no-package-lock
 # Install Deployment Microservice Modules!
 RUN cd /telescope/tools/autodeployment && npm install --no-package-lock --ignore-scripts
 
+# -------------------------------------
+# Context: Git Repo Handling
+FROM alpine/git as git
+
+WORKDIR "/telescope"
+
+COPY .git .git
+
+RUN git reflog expire --expire=now --all
+RUN git repack -a -d --depth=1 --window=10
+RUN git gc --prune=now
```

Comment:
I wonder if instead of messing with the `.git` here, we should just do the `--depth=1` here https://github.com/Seneca-CDOT/telescope/blob/master/tools/autodeployment/deploy_stage.sh#L16, so it does the right thing for staging/prod.  What happens locally I don't care about optimizing.

Reply:
Great idea. Updated.